internal/trial/trial.go

```diff
@@ -90,6 +91,38 @@ func IsTrialJobReference(t *redskyv1beta1.Trial, ref *corev1.ObjectReference) bo
 	return true
 }
 
+// IsBaseline checks to see if the supplied trial is a baseline for an experiment.
+func IsBaseline(t *redskyv1beta1.Trial, exp *redskyv1beta1.Experiment) bool {
```

Comment:
Dont we do something similar to this elsewhere?

Reply:
The closest thing is [`server.FromCluster`](https://github.com/thestormforge/optimize-controller/blob/master/internal/server/server.go#L45) however that generates an API object so instead of `IntOrStr` (which we can do a Go `==` check on) it has `NumberOrString`.